src/extension/src/azure/azure-app-service/appServiceProvider.ts

```diff
@@ -65,9 +65,12 @@ export class AppServiceProvider {
         selections.subscriptionItem
       );
       this.writeARMTemplatesToApp(appPath, template, parameters);
+      const deploymentName = (
+        selections.siteName + APP_SERVICE_DEPLOYMENT_SUFFIX
+      ).substr(0, CONSTANTS.DEPLOYMENT_NAME.MAX_LENGTH);
```

Comment:
constraint of deployment name is 64 characters that's why I'm trimming it (also, since deployment names are not very visible for users, I just did a simple trim -- I could trim the project name, leave the timestamp in tact, but I just feel like that's too much logic for generating a deployment name)